app/models/audit.rb

```diff
@@ -13,7 +13,13 @@
 class Audit < ActiveRecord::Base
   include Uuidable
 
-  PAST_TENSES = { 'scan' => 'Scanned', 'destroy' => 'Destroyed' }
+  PAST_TENSES = {
+    'scan' => 'Scanned',
+    'destroy' => 'Destroyed',
+    'Uploaded from manifest' => 'Uploaded from manifest',
+    'removed all labwares' => 'Removed all labwares',
```

Comment:
Not blocking, but not sure if the key is meant to be present, and value past?  'Uploaded from manifest' => 'Upload from manifest' maybe?